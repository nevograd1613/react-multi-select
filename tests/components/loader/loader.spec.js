import React from "react";
import ShallowRenderer from "react-shallow-renderer";

import Loader from "../../../src/components/loader/loader";

describe("Loader", () => {
  test("default snapshot", () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(<Loader />);
    expect(tree).toMatchSnapshot();
  });
});
