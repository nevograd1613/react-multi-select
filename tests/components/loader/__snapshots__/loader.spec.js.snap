// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader default snapshot 1`] = `
<div
  className="loader"
>
  <WithStyles(CircularProgress)
    style={
      {
        "width": 100,
      }
    }
  />
</div>
`;
